### DISCUSSION TITLE
HELP!

### DISCUSSION THREAD

**Discussion by RedRavenYouTube** [LBPUnion/ProjectLighthouse#610] [Q&A] [answered]
I was on beacon.lbpunion.com and I tried to log in but it wouldn't let me.

**Comment by sudokoko** [comment, marked as answer]
You need to be selected for LBP Union's USC Explorer program before you can gain access. Head [here](https://discord.gg/lbpunion) and navigate to the "settings" channel and get yourself the USCEnlist role. We resume random selections for testers in January.

**Reply by RedRavenYouTube** [reply]
Will the register be on the main website when on public beta/release?

**Reply by sudokoko** [reply]
> Will the register be on the main website when on public beta/release?

I'd assume so.

**Comment by m88youngling** [comment]
Closing this. This repository is for development discussion. Please direct all general questions to our site forum at https://www.lbpunion.com/forum
